packages/valory/skills/abstract_round_abci/base.py

```diff
@@ -1518,7 +1516,7 @@ def process_payload(self, payload: BaseTxPayload) -> None:
         if self.keeper_sent_payload:
             raise ABCIAppInternalError("keeper already set the payload.")
 
-        self.keeper_payload = getattr(payload, self.payload_attribute)
+        self.keeper_payload = payload
```

Comment:
We break backwards compatibility here as there's no benefit from maintaining it and the upgrade is simple.